data_preprocessing/name_preprocessing.py

```diff
@@ -0,0 +1,230 @@
+"""
+Contains helper functions:
+Create and pickles a People Database from CSV
+Extract author and recipient aliases from CSV with documents info
+Involve taking the documents (originally from CSV) and extract authors, organizations,
+and recipients
+Also involve converting organization names to their official, clean names
+"""
+import time
+import json
+from pathlib import Path
+import unittest
+from collections import Counter
+import pandas as pd
+from person import Person
+from people_db import PeopleDatabase
+from clean_org_names import RAW_ORG_TO_CLEAN_ORG_DICT
+
+
+def merge_names_from_file(name_file, out_file):
+    """
+    Creates a people db from reading json file (dict of raw names and counts) and merges people
+    in it. Stores people db in a pickle file
+    :param name_file: Path to json file
+    :param out_file: Path for output pickle file of the created PeopleDB
+    :return:
+    """
+
+    with open(name_file, 'r') as infile:
+        name_dict = json.load(infile)
+
+    initial_time = time.time()
+
+    # add everyone to a PeopleDatabase
+    people_db = PeopleDatabase()
+    for name in name_dict:
+        if name_dict[name] >= 3:
+            people_db.add_person_raw(name_raw=name, count=name_dict[name])
+
+    print("Length: ", len(people_db))
+
+    # then merge the duplicate / similar names
+    people_db.create_positions_csv()
+    people_db.merge_duplicates()
+
+    people_db.store_to_disk(out_file)
+    print("Merging names took", time.time() - initial_time)
+
+
+
+def parse_doc_metadata_csv(csv_path, people_db=None):
```

Comment:
I refactored add_au_and_rc_org into this function. 
It now works without a previous people_db and treats recipients and authors the same way, which made the processing simpler.